docs/quickstart.rst

```diff
@@ -222,7 +222,24 @@ amount of requests you wish to send.
 
     // Force the pool of requests to complete.
     $promise->wait();
+    
+Or using a closure that will return a promise once the pool calls the closure.
 
+.. code-block:: php
+
+    $client = new Client();
+
+    $requests = function ($total) use($client) {
```

Comment:
Please add a space between `use` and `($client)` for consistency with other Guzzle documentation and code. Same with line 235.